index.js

```diff
@@ -197,11 +197,17 @@ Plaid.Client.prototype.deleteInfoUser =
 
 
 // exchangeToken
-Plaid.Client.prototype.exchangeToken = function(public_token, callback) {
+Plaid.Client.prototype.exchangeToken =
+  function(public_token, account_id, callback) {
+  if (typeof account_id === 'function') {
+    callback = account_id;
+    account_id = null;
+  }
```

Comment:
I'd prefer to define a second method, `exchangeTokenWithAccountId`, rather than juggle arguments. `exchangeToken` could then be defined in terms of `exchangeTokenWithAccountId` (by providing `null` as `account_id`).

Others may not share this preference, of course. :)